_layouts/bib.html

```diff
@@ -25,6 +25,16 @@
               {% assign author_is_self = true %}
             {% endif %}
           {% endif %}
+          {% assign coauthor_url = "" %}
```

Comment:
Minor stylistic change:
```suggestion
          {% assign coauthor_url = nil %}
```